packages/components/src/StatusLabel/StatusLabel.mdx

```diff
@@ -0,0 +1,108 @@
+---
+name: StatusLabel
+menu: Components
+route: /components/status-label
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { StatusLabel } from ".";
+import { Content } from "../Content";
+
+# Status Label
+
+The `<StatusLabel>` component is a visual component that allows users to quickly
+determine the status of an item (e.g., "Active", "Overdue").
+
+```ts
+import { StatusLabel } from "@jobber/components/StatusLabel";
+```
+
+<Playground>
+  <StatusLabel label="Bob" />
+</Playground>
+
+## Props
+
+<Props of={StatusLabel} />
+
+## Usage Guidelines
+
+StatusLabel is a more opinionated version of
+[InlineLabel](/components/inline-label) and offers only 5 possible status
+representations:
+
+### Success
+
+Convey that an item is in a successful state.
+
+### Critical
+
+Alert the user to a critically important issue.
+
+### Warning
+
+Warn the user of a potential forthcoming issue.
+
+### Informative
+
+Inform the user about something that may not require action.
+
+### Inactive
+
+Signify that an item has been archived, closed, or otherwise removed from an
+active workflow.
+
+<Playground>
+  <Content>
+    <StatusLabel label="Success" status="success" alignment="start" />
+    <StatusLabel label="Critical" status="critical" alignment="start" />
+    <StatusLabel label="Informative" status="informative" alignment="start" />
+    <StatusLabel label="Warning" status="warning" alignment="start" />
+    <StatusLabel label="Inactive" status="inactive" alignment="start" />
+  </Content>
+</Playground>
+
+### Alignment
+
+Align the color indicator with the start or end of a layout as needed.
+
+For example, in a [List](/components/list) where the StatusLabel is on the
+right, use the `end` alignment.
+
+<Playground>
+  <div style={{ display: "flex", justifyContent: "space-between" }}>
+    <StatusLabel label="Start" status="inactive" alignment="start" />
+    <StatusLabel label="End" status="inactive" alignment="end" />
+  </div>
+</Playground>
+
+## Related Components
+
+- [InlineLabel](/components/inline-label) should be replaced with StatusLabel
+  where possible to provide a more intentional communication of status to users
+
+## Content Guidelines
+
+StatusLabel does not accept any child content. It presents only a text-based
+label describing the status, and a visual color indicator to reinforce the
+status.
+
+The label should be short, ideally no longer than two words.
+
+## Accessibility
+
+The label portion of StatusLabel should be readable by assistive technology.
+
+The color indicator is supplemented by the label to ensure that users who cannot
```

Comment:
This was a little hard for me to read and I think can use some simpler wording? Something like "Color indicator is always used with a text label so that color is not the only method of communicating status"? 🤷‍♀️